app/application/route.js

```diff
@@ -0,0 +1,68 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class ApplicationRoute extends Route {
+  @service store;
+
+  async beforeModel() {
+    const user1 = {
+      id: 1,
+      username: 'admin',
+      password: 'admin123',
+      email: 'admin@admin.com',
+      isRequestor: true,
+    };
```

Comment:
For avoid overriding these records everytime when you open app, add this - your 'database' will be spread just when you clean all localstorage: 
```suggestion
    const isFilledDatabase = localStorage.length;

    if (isFilledDatabase) {
      return;
    }
    
    const user1 = {
      id: 1,
      username: 'admin',
      password: 'admin123',
      email: 'admin@admin.com',
      isRequestor: true,
    };
```